portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,149 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let eerieModeOn = false; 
+
+const root = document.documentElement;
+
+const secretText = document.querySelector("#secret-text"); 
+const mePic = document.querySelector("#me-pic"); 
+const tomiePic = document.querySelector("#tomie-pic"); 
+const mainHeading = document.querySelector("#main-heading"); 
+
+//fade in main text
+const mainText = document.querySelector(".main-text");
+window.addEventListener('load', e => {
+    mainText.classList.add("fade-text-in"); 
+    extraContent.style.display = "none"; 
+})
+
 /**
- * Adds a random greeting to the page.
+ * Keeps track of mouse and updates our special, circle pointer to
+ * follow cursor movement. 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+const circleAroundMouse = document.querySelector(".cursor");
+window.addEventListener('mousemove', e => {
+    circleAroundMouse.style.top = e.pageY + "px"; 
+    circleAroundMouse.style.left = e.pageX + "px"; 
+})
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+const secretMessages = ["...", "....", "Would you like to see something fun?"]; 
+let messageIndex = 0; 
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+/**
+ * Changes the theme of the page 
+ */
+const extraContent = document.querySelector("#extra-content");
+function activateEerieMode() {
+    eerieModeOn = true; 
+    mePic.classList.add("fade-image-out"); 
+    document.body.style.backgroundColor = "black"; 
+    root.style.setProperty('--target-color', "red");
+    root.style.setProperty('--circle-cursor-color', "red"); 
+    root.style.setProperty('--highlighted-text-color', "white"); 
+    mainHeading.classList.add("eerie-text"); 
+    sections.forEach(section => {
+        section.classList.add("eerie-text"); 
+    });
+    mainHeading.textContent = "Something Fun"; 
+    mainText.textContent = mainText.dataset.reversed; 
+    mainText.classList.add("eerie-text"); 
+    //the following lines display extra, EM-exclusive features 
+    extraContent.style.display = "block"; 
+    extraContent.style.opacity = "1";  
+    //TODO: fix below line, which doesn't transition the opacity
+    extraContent.classList.add("change-extra-content-opacity"); 
```

Comment:
yea I couldn't actually find extra-content on the page :(

Reply:
Just to clarify - did the "movie recommendation" dropdown not appear when "eerie mode" was switched on?